manifests/s/Sky/SkyGo/21.8.2.0/Sky.SkyGo.locale.en-GB.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Sky.SkyGo
+PackageVersion: 21.8.2.0
+PackageLocale: en-GB
+Publisher: Sky
+PublisherUrl: https://www.sky.com/
+PublisherSupportUrl: https://www.sky.com/help/home
+PrivacyUrl: https://www.sky.com/help/articles/sky-privacy-and-cookies-notice#get-in-touch
+Author: Sky UK Limited
+PackageName: Sky Go
+PackageUrl: https://www.sky.com/watch/sky-go/
+License: Propiatory
+ShortDescription: Sky Go is an online television service from Sky Group
+Description: The Sky Go desktop app makes it even easier for you to find the programmes you want to watch. And with Sky Mobile, you can watch Sky TV on the go without using your data.
+Moniker: Sky Go

```

```suggestion
Moniker: sky-go
```
It's better to use hyphens rather than spaces here.